aea/decision_maker/base.py

```diff
@@ -301,11 +306,37 @@ def _handle_tx_message(self, tx_message: TransactionMessage) -> None:
         :param tx_message: the transaction message
         :return: None
         """
-        score_diff = self.preferences.get_score_diff_from_transaction(self.ownership_state, tx_message)
-        if score_diff >= 0.0:
-            pass  # TODO: implement tx flow
-            # envelope = Envelope()
-            # self.outbox.put(envelope)
+        crypto_identifier = CURRENCY_TO_ID_MAP.get(cast(str, tx_message.get("currency")))
+        amount = cast(float, tx_message.get("amount"))
+        crypto_object = self._wallet.crypto_objects.get(crypto_identifier)
+        if self._is_acceptable_tx(crypto_object, amount):
```

Comment:
for now we accept any transaction which is affordable with no regard for preferences.